fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
@awoods Using a README would certainly be my preference over inline comments. To avoid duplication and the inevitable drift in consistency, it might be best to just include a link in the README to the recommendations which are captured in the wiki, once those are written. This would seem to fit nicely into the follow-on ticket you suggested in fcrepo-2402.